tests/cephfs/cephfs_utilsV1.py

```diff
@@ -110,6 +112,257 @@ def get_fs_info(client, fs_name="cephfs"):
+                    sudo=True,
+                    cmd="cephadm shell -- ceph --admin-daemon /var/run/ceph/ceph-mds.%s."
+                    "asok %s| grep path" % (active_mds_node_2, val),
+                )
+                return (
+                    out_1.rstrip("\n"),
+                    out_2.rstrip("\n"),
+                    0,
+                )
+
+            elif val == "session ls":
+                out_1, err_1 = active_mds_node_obj_1.exec_command(
+                    sudo=True,
+                    cmd="cephadm shell -- ceph --admin-daemon /var/run/ceph/ceph-mds.%s."
+                    "asok %s" % (active_mds_node_1, val),
+                )
+                out_2, err_2 = active_mds_node_obj_2.exec_command(
+                    sudo=True,
+                    cmd="cephadm shell -- ceph --admin-daemon /var/run/ceph/ceph-mds.%s."
+                    "asok %s" % (active_mds_node_2, val),
+                )
+                return (
+                    out_1.rstrip("\n"),
+                    out_2.rstrip("\n"),
+                    0,
+                )
+
+    @staticmethod
+    def auto_evict(client_node, clients, rank):
+        """
+        Evict Client from the ceph cluster
+        Args:
+            client_node:
+            clients:
+            rank:
+
+        Returns:
+
+        """
+        grep_pid_cmd = """sudo ceph tell mds.%d client ls | grep '"pid":'"""
+        out, rc = client_node.exec_command(cmd=grep_pid_cmd % rank)
+        client_pid = re.findall(r"\d+", out)
+        while True:
+            for client in clients:
+                try:
+                    for pid in client_pid:
+                        client.exec_command(
+                            sudo=True, cmd="kill -9 %s" % pid, container_exec=False
+                        )
+                        return 0
+                except Exception as e:
+                    print(e)
+                    pass
+
+    @staticmethod
+    def manual_evict(client_node, rank):
+        """
+        Manual evict Client from the ceph cluster
+        Args:
+            client_node:
+            rank:
+
+        Returns:
+
+        """
+        grep_cmd = """ sudo ceph tell mds.%d client ls | grep '"id":'"""
+        out, rc = client_node.exec_command(cmd=grep_cmd % rank)
+        client_ids = re.findall(r"\d+", out)
+        grep_cmd = """ sudo ceph tell mds.%d client ls | grep '"inst":'"""
+        log.info("Getting IP address of Evicted client")
+        out, rc = client_node.exec_command(cmd=grep_cmd % rank)
+        op = re.findall(r"\d+.+\d+.", out)
+        ip_add = op[0]
+        ip_add = ip_add.split(" ")
+        ip_add = ip_add[1].strip('",')
+        id_cmd = "sudo ceph tell mds.%d client evict id=%s"
+        for client_id in client_ids:
+            client_node.exec_command(cmd=id_cmd % (rank, client_id))
+            break
+
+        return ip_add
+
+    @staticmethod
+    def osd_blacklist(client_node, ip_add):
```

Comment:
Should we change the name to "osd_blacklist_rm_client" or something more appropriate as we are removing client ip from blacklist?